content/en/getting-started/installation.md

```diff
@@ -27,6 +27,22 @@ All of these tools must be in your PATH, so that kluctl can easily invoke them.
    curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/kluctl-linux-amd64"
    ```
 
+2. **Optional**: Validate the kluctl binary *only available as of version >= v2.7.0*
+
+   Download the kluctl checksum file:
+   ```bash
+   kluctl_version=$(curl --silent -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kluctl/kluctl/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+   curl -LO "https://github.com/kluctl/kluctl/releases/download/${kluctl_version}/checksums.txt"
+   ```
+   Validate the kluctl binary against the checksum file:
+   ```
+   echo $(grep kluctl-linux-amd64 checksums.txt) | sha256sum --check
```

Comment:
This line doesn't work for me. Could you check, if there is something missing. @matzegebbe 

Output of the checksums.txt
```
0df45073be51080031febf93afc2cf95f96c68910a8ca5e2db6ed1775412bd1d  kluctl_2.7.7_Linux_arm64.tar.gz
7013966f55c6fb46f109fc1916b4b49f84b0fb66eb1dab283fa0c641615a49c9  kluctl_2.7.7_Darwin_arm64.tar.gz
7c2c6d2ad0b3629323f6d637ccf91ee70d3ad55753b59997536bcb69241f2ec5  kluctl_2.7.7_Windows_x86_64.tar.gz
bdc7235c706dd0dabb3af87f5d12821f89de35903a00aa36d78191fa457d7610  kluctl_2.7.7_Linux_x86_64.tar.gz
f7ae9fecb6b55c142b538e621685780f902a9ea6e0b74113524a8d71f2a1ac7d  kluctl_2.7.7_Darwin_x86_64.tar.gz
```